packages/governance/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "maxNodeModuleJsDepth": 2,
   },
   "include": [
-    "exported.js",
+    // omit exported.js because 1) it's empty and would overwrite exported.d.ts
+    // and 2) because it's only for consumption in other packages
```

Comment:
I'm a little confused.
"for consumption in other packages" seems like a reason to include it, not a reason to omit it.

Does `exported.d.ts` suffice somehow when other packages want `exported.js`?

Reply:
that it's used in other packages is a reason to include it _in package.json `files`_. but this is the tsconfig for typechecking this package, so it's superfluous. and because there's a `.d.ts` it needn't be part of the build. I'll clarify